docs/fileio_usage.rst

```diff
@@ -0,0 +1,62 @@
+=====
+Usage
+=====
+
+The easiest way to use :py:mod:`orsopy.fileio` (the module of :py:mod:`orsopy` that includes file reading and writing) to produce **metadata-rich .ort reduced reflectometry files** involves integrating this into your data reduction workflow.
+Early in the workflow, the :py:mod:`orsopy.fileio` should be imported and an empty :code:`orsopy.fileio.orso.Orso` header object (here we also import :py:mod:`numpy` which will be used later). 
+
+.. code-block:: python 
+
+    import numpy as np
+    from orsopy.fileio.orso import Orso
+
+    header = fileio.orso.Orso.empty()
+
+Having created the empty header object we can start to populate the appropriate components of it. 
+It is generally a good idea to populate the components as particular steps occur in the reduction process. 
+For example, if we want to identify the probing radiation as neutrons, we include this as follows. 
+
+.. code-block:: python 
+
+    header.data_source.experiment.probe = 'neutrons'
```

Comment:
I think the best practice would not be to modify the empty Orso but to just use it as reference to see what is needed but to build the file directly from the classes. 
Direct assignment of attributes could be problematic as no check of correctness is done.

Reply:
This is a good point. Let me revise the example.

Reply:
When is a check for correctness done?

Reply:
![Screenshot 2022-10-18 at 09 12 58](https://user-images.githubusercontent.com/15774281/196361449-4d1378d1-a955-4f1a-bcd6-1fd4dbf223f3.png)

I think he means this